docs/api/env/env.md

```diff
@@ -540,12 +549,32 @@ A wrapper class around py-evm which provides a "contract-centric" API. More deta
     - `gas`: The gas limit provided for the execution (a.k.a. `msg.gas`).
     - `value`: The ether value to attach to the execution (a.k.a `msg.value`).
     - `data`: The data to attach to the execution (a.k.a. `msg.data`).
+    - `simulate`: If True, the call is executed in a context that is rolled back after execution.
 
     ---
 
     **Returns**
 
-    The return value from the top-level call.
+    A `ComputationAPI` object containing the execution result. Key attributes include:
+    - `output`: The return data as bytes
+    - `is_error`: Boolean indicating if the execution failed
+    - `error`: The exception if execution failed
+    - `gas_used`: Amount of gas consumed
+
+    ---
+
+    **Important**
+
+    Unlike `execute_code`, if the computation fails (`is_error` is True), this method raises the error as an exception.
```

Comment:
That behavior seems a bit unintuitive, especially given that it needs a note. Would it make sense to make `raw_call` return bytes, like in Vyper, and leave `execute_code` as a low-level version? Might also need a third function for completeness, one that does not throw and returns ComputationABI but does not require passing bytecode...

We don't need to deal with that now ofc